docs/tutorials/cloud_fit_deploy/autopilot-autogluon.md

```diff
@@ -0,0 +1,4 @@
+# New managed AutoGluon-Tabular experience on Amazon SageMaker Autopilot
+:label:`autopilot_autogluon`
+
+[Checkout managed AutoGluon-Tabular experience on Amazon SageMaker Autopilot](https://aws.amazon.com/blogs/machine-learning/amazon-sagemaker-autopilot-is-up-to-eight-times-faster-with-new-ensemble-training-mode-powered-by-autogluon/)
```

Comment:
Add a brief 4-5 sentence overview of what AutoPilot is, maybe just copy-paste from their own page, just to give some context to the user.